From 22d7a9050b04b021e33430d0f7e3c5ee80c24caa Mon Sep 17 00:00:00 2001
From: Peter Seiderer <ps.report@gmx.net>
Date: Tue, 17 Dec 2019 11:01:59 +0100
Subject: [PATCH] Disable designer tool (fixes configure error).

Fixes:

  .../build/qt5tools-5.14.0/src/designer/src/components/lib/lib.pro
  Project ERROR: Multiple modules claim plugin type 'designer' (uitools, in addition to designer)

Signed-off-by: Peter Seiderer <ps.report@gmx.net>
[Rebased for Qt5.15.0]
Signed-off-by: Peter Seiderer <ps.report@gmx.net>
[Rebased for Qt5.15.1]
Signed-off-by: Peter Seiderer <ps.report@gmx.net>
[Rebased for Qt5.15.2]
Signed-off-by: Peter Seiderer <ps.report@gmx.net>
---
 src/linguist/linguist/linguist.pro | 2 +-
 src/src.pro                        | 5 +----
 2 files changed, 2 insertions(+), 5 deletions(-)

diff --git a/src/linguist/linguist/linguist.pro b/src/linguist/linguist/linguist.pro
index d083896..7d74cfc 100644
--- a/src/linguist/linguist/linguist.pro
+++ b/src/linguist/linguist/linguist.pro
@@ -1,4 +1,4 @@
-QT += core-private gui-private widgets uitools-private printsupport
+QT += core-private gui-private widgets printsupport
 
 DEFINES += QT_NO_CAST_FROM_ASCII QT_NO_CAST_TO_ASCII
 
diff --git a/src/src.pro b/src/src.pro
index f2675c7..360de50 100644
--- a/src/src.pro
+++ b/src/src.pro
@@ -6,12 +6,9 @@ qtHaveModule(widgets) {
     } else {
         QT_FOR_CONFIG += widgets
         qtConfig(pushbutton):qtConfig(toolbutton) {
-            SUBDIRS = designer \
-                      pixeltool
+            SUBDIRS = pixeltool
 
             !static|contains(QT_PLUGINS, qsqlite): SUBDIRS += assistant
-
-            linguist.depends = designer
         }
         qtHaveModule(quick):qtConfig(thread):qtConfig(toolbutton): SUBDIRS += distancefieldgenerator
     }
-- 
2.29.2

